test/seekrets.bats

```diff
@@ -60,9 +60,9 @@ load test_helper
     [ $status -gt 0 ]
 }
 
-@test "git-seekrets does not find newrelic keys as aws keys in test repo" {
```

Comment:
Wait, I thought the issue was that a new relic key was tripping the AWS rule, hence the unmatch needed in the AWS rule?